feeds/lv.json

```
@@ -10,18 +10,26 @@
             "name": "pieturas",
             "type": "http",
             "url": "https://gsvalbe.id.lv/pieturas/dati/latvia.zip",
+            "license": {
+                "spdx_identifier": "CC0-1.0"
+            },
             "fix": true
         },
         {
             "name": "pieturas",
             "type": "url",
             "spec": "gtfs-rt",
             "url": "https://gsvalbe.id.lv/pieturas/dati/gtfs-rt.pb"
+            "license": {
+                "spdx_identifier": "CC0-1.0"
```

```suggestion
                "spdx-identifier": "CC0-1.0"
```
Please consult https://transitous.org/doc/ when you are unsure about the naming of the options.

---

My bad, I've pasted it from other project and didn't realise it's different